### ISSUE TITLE
Font missing diacritics characters

### ISSUE THREAD

**Issue by sq3tle** [go-vikunja/vikunja#143]
### Description

Font used in sidebar and task editing panel is missing diacritics characters used in Polish language. In case of some languages, it may be worth to fallback to some regular font.

### Vikunja Frontend Version

0.20.5

### Vikunja API Version

0.20.4

### Browser and version

chrome 112.0.5615.138

### Can you reproduce the bug on the Vikunja demo site?

Yes

### Screenshots

![image](https://user-images.githubusercontent.com/58919648/233987231-9df85e36-e5d7-473f-9b55-e0bfa3b73138.png)

**Comment by orhid** [comment]
has there been any movement on this bug?

**Comment by dpschen** [comment]
> has there been any movement on this bug?

No, but I try to fix this soon (will try this week). Since our internationalisation has progressed a lot this became more urgent.

**Comment by sceptri** [comment]
Hey, just wanted to ask... any update on this bug?

**Comment by kolaente** [comment]
I've adjusted this in https://github.com/go-vikunja/vikunja/commit/aea2f708d34ef0b0a87caefaeb842a1c0bbf1b83 - please check with the next unstable build (should be ready for deployment in ~30min, also on [try](https://try.vikunja.io)).

@dpschen This was the fastest way to fix the problem here, feel free to optimize this further.

**Comment by dpschen** [comment]
All good. Sry that i didn’t get to this yet.

@kolaente I‘m a bit confused that you didn’t need to change the unicode range.

**Comment by kolaente** [comment]
@dpschen I did: https://github.com/go-vikunja/vikunja/commit/3b8258d57ed011a00bb057ed9394f13418246da6
